### DISCUSSION TITLE
Erreur lors de l'ajout de l'intégration

### DISCUSSION THREAD

**Discussion by jcl-concept** [jmcollin78/versatile_thermostat#279] [Q&A]
Bonjour,

J'ai une erreur lorsque j'ajoute l'intégration Versatile Thermostat aussi bien en version 4.3 que la toute nouvelle 5.0 :
Le flux de configuration n'a pas pu être chargé. 500 Internal Serveur Error

J'ai cette erreur dans le fichier homeassistant.log :
AttributError: type objectif 'VersatileThermostatAPI' has no attribute '_hass'
In file
/Config/custom_components/versatile_thermostat/Vtherm_api.py, Line 31,....

Pourriez vous m'aider a débloquer l'installation ?

Merci par avance
Jean-Christophe 

hass core : 2023.12.3
Hass supervisor : 2023.11.6
OS : 11.2
Interface : 20231208.2

**Comment by jmcollin78** [comment]
Hello, @jcl-concept ,

J'aurai besoin de plus de log stp. Peux-tu passer en mode debug, redémarrer et m'envoyer les logs autour de cette erreur. Merci d'avance.

Ca a changé en 5.0 en effet. Ce que je comprends pas c'est que ca ne marchait pas en 4.3. Tu as certainement un problème d'installation.

**Reply by jcl-concept** [reply]
Bonsoir @jmcollin78 ,

Merci pour votre réponse. 

Cela pourrait être un problème d'installation du composant ? Je l'ai installé via HACS et non en manuel.
Il y a des prérequis avant l'installation ?

Voici les logs de la version 4.3.0 après avoir activé le niveau debug dans le confguration.yaml
`logger:
  default: warning
  logs:
    custom_components.versatile_thermostat: debug`

[home-assistant_versatile_thermostat.log](https://github.com/jmcollin78/versatile_thermostat/files/13708307/home-assistant_versatile_thermostat.log)

**Comment by jmcollin78** [comment]
Hello, @jcl-concept ,

C'est corrigé en 5.0.1. Y avait un bug sur une fresh installation.

**Reply by jcl-concept** [reply]
Bonsoir,

Je confirme, cela fonctionne à présent 👍.

Merci pour votre réactivité. Je me lance dans l'intégration avec des radiateurs fil pilote à présent.

Cdlt,
